test/markup/javascript-repl/sample.txt

```diff
@@ -0,0 +1,6 @@
+> a=1
+1
+> b=2
+2
+> 
+> 
```

Comment:
This needs a matching `test/markup/javascript-repl/sample.expect.txt` file... you can run the markup tests after you've built the library with:

```
npm run build && npm run test-markup
```

Reply:
And please add at least one `...` line as well.